protocol/pushnotificationclient/client.go

```diff
@@ -1245,6 +1307,48 @@ func (c *Client) handleAllowedKeyList(publicKey *ecdsa.PublicKey, allowedKeyList
 	return ""
 }
 
+func (c *Client) MyPushNotificationQueryInfo() ([]*protobuf.PushNotificationQueryInfo, error) {
```

Comment:
Does the "My" from `MyPushNotificationQueryInfo` refer to the node's info rather than a peer's info?

I know there are a lot of references to "my" and "their", are these placeholder terms? I kind of like the use of "my and "their" but we also have the concept of "node" and "peer".

Does "my" and "their" have a different meaning / use case than "node" and "peer"?

Reply:
In this case it refers to the identity you started the `messenger` with, which is not equivalent to node nor peer I think.

My understanding is that node generally refers to the `geth` node started, while peer generally refers to the `waku` peers connected.

In this cases is neither, as the identity might change, but the peer will stay the same (we login with a different account).

What about `CurrentIdentityPushNotificationInfo`?

Reply:
Ok, that is a reasonable distinction. I'm imagining what a future newcomer would think when looking at this code in the future.

`CurrentIdentityPushNotificationInfo` I think is more descriptive, I would say it is a better choice.

Does "their" refer to PN peers?